### PR TITLE
Added azcopy feedstock

### PR DESCRIPTION
**Pull request by krisevans**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/azcopy/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://azcopyvnext.azureedge.net/release{{ release_date }}/azcopy_{{ platform }}_amd64_{{ version }}.{{ ext }}
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: "cp $SRC_DIR/azcopy $PREFIX/lib/azcopy"  # [not win]
```
Comment (dhirschfeld) [inline recipes/azcopy/meta.yaml:23]: I think this should also be `bin`?
```suggestion
  script: "cp $SRC_DIR/azcopy $PREFIX/bin/azcopy"  # [not win]
```

File: recipes/azcopy/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  script: "echo F | xcopy %SRC_DIR%\\azcopy.exe %PREFIX%\\bin\\azcopy.exe /F /Y"  # [win]
+
+requirements:
+  build:
+  host:
+
+test:
+  commands:
+    - test -f $PREFIX/lib/azcopy  # [unix]
+    - if not exist %PREFIX%\\bin\\azcopy.exe exit 1  # [win]
```
Comment (dhirschfeld) [inline recipes/azcopy/meta.yaml:33]:
```suggestion
    - azcopy -h
```

File: recipes/azcopy/meta.yaml (hunk 1 above)
Comment (krisevans) [reply recipes/azcopy/meta.yaml:23]: I tried that first but it didn't work

**Comment by dhirschfeld** [comment]
Will need to be built with the `cos7` container for linux so have skipped for now and can fix in the feedstock.

**Review by dhirschfeld — APPROVED** [review]

**Comment by dhirschfeld** [comment]
ping @conda-forge/staged-recipes - any objection to merging this repackaging of the AzCopy binary?

**Pull request merged**